### PR TITLE
Rename Layout Directory, update SCSS font naming

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. /layouts is renamed to /regions
2. site.jsx is moved to /ui directory
3. Font variables are renamed
4. Update routes

### CODE REVIEW

**Comment by kylecesmat** [comment]
:m:

**Comment by kylecesmat** [comment]
:star2:

**Comment by natestormer** [comment]
:+1:

**Comment by natestormer** [comment]
:star2:

File: application/ui/scss/components/typography/_grouping-elements.scss [hunk 1]
```diff
@@ -5,10 +5,14 @@
 
 // *******************************************
 // *******************************************
+//
+//
```
Comment (canfie1d) [inline application/ui/scss/components/typography/_grouping-elements.scss]: delete lines

**Comment by kylecesmat** [comment]
:star2:

**Comment by canfie1d** [comment]
:+1:

**Pull request merged**